ow_lander/action/ARM_MOVE_JOINT.action

```diff
@@ -0,0 +1,10 @@
+# goal definition
+bool relative
+int32 joint
+float32 angle
+---
+# result
+float32[] final_angle    # joint final positions
```

Comment:
This seems fine, though I added a comment to the unification document that we should specify the ROS type of Float32 or Float64.